lessons/unittest/lesson.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: 'Unittesting in Python'
+visible: true
+---
+
+# Brief introduction to the Python unittest module
+
+ - **Authors**: Andre R. Erler
+ - **Research field**: Atmospheric Physics
+ - **Lesson topic**: Unittesting (software development)
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/unittest>
+
+The [unittest](https://docs.python.org/2/library/unittest.html) module primarily provides a class TestCase to facilitate the writing of tests for "units" of code inside a larger project.
+Larger software projects benefit from unit tests, because smaller units of the software can be tested individually and checked for errors before they are added to the main project. After a project has grown over time, a suite of unittests can also be very helpful in debugging, because the source of an error can quickly be located.
+
+Typically test cases are implemented as child classes of the `unittest.TestCase` class. The methods `TestCase.setUp` can be overloaded to provide common setup procedures for all test methods; the method `TestCase.tearDown` can be used for cleanup; both methods are executed before (after) each test. All methods of the form `TestCase.test*` are considered test cases and will be executed by `unittest`, if `unittest.main()` is called.
+
+To run several tests suites at once, the command line utility `nose` can be used: it scans all Python scrips in the current directory and executes all available test cases.
```

Comment:
Does this require any action from me? Or do you make this change? (I'm not familiar with the protocol here)